### DISCUSSION TITLE
empty response from server

### DISCUSSION THREAD

**Discussion by yecheng0324** [BurnHair/docs#170] [技术支持]
昨天用NextChat 让4o给我改个脚本，前面都正常，下午5点开始一直返回如下图：
今早登录页面，看到了清额度的公告，昨天下午额度应该还在的。
我去充值了，在让输出才正常

 
<img width="472" alt="image" src="https://github.com/user-attachments/assets/68d8dda5-458d-455b-a3e7-e73bb56174c3">

**Comment by BennyThink** [comment]
你可能是符合清理规则的用户，所以在昨天凌晨的时候被清了

**Reply by yecheng0324** [reply]
凌晨被清，我看了公告是符合规则的。
出这个错误提醒的时候是下午应该那时候还没清吧

**Reply by BennyThink** [reply]
啊，那你开的新的会话试试看呢？

**Reply by yecheng0324** [reply]
当时是不行的，今早才可以的
